### PR TITLE
Dip 158 remove ready for settlement

### PR DESCRIPTION
**Pull request by YinonFirstDAG**
- Removed ReadyForSettlement command
- Added expectation for the Merchant VASP to find transaction on the blockchain
- Added abort codes for the AbortPayment command
- Added abort sequences (Missing Recipient Signature, Could Not Put Transaction On-Chain)
- Added appendix about including the reference ID in the transaction metadata (including reference to DIP-10)
- Removed sequence diagram temporarily (until we upload the final image to github and embed from there)

### CODE REVIEW

File: dips/dip-158.md
Comment (dimroc) [inline dips/dip-158.md:121]: Let's wait until we have a sequence diagram before accepting/merging.

File: dips/dip-158.md
Comment (davidiw) [inline dips/dip-158.md:73]: should we use customer VASP and merchant VASP instead of sender VASP and receiver VASP?

**Review by davidiw — COMMENTED** [review]
overall looks fine, should we use terminology like merchant VASP and customer VASP instead of sender or receiver? Except in the case where the the role is independent of the customer / merchant VASP?

File: dips/dip-158.md
Comment (YinonFirstDAG) [reply dips/dip-158.md:9]: I will wait for DIP-183 to land and then update this section and links

File: dips/dip-158.md
Comment (YinonFirstDAG) [reply dips/dip-158.md:73]: The glossary section covers that, I think:
>>>
For the sake of simplicity, we use the following terms:
* "Sender", "Payer", "Wallet" or "customer VASP" - The customer's wallet. It is the VASP representing the customer. This is the VASP that will send the funds and eventually will submit the on-chain transaction to the blockchain
* "Receiver", "Receiving VASP" or "Merchant VASP" - The VASP representing the merchant which receives the funds (transaction)
>>>
We previously agreed to use sender/receiver throughout the DIP.

File: dips/dip-158.md
Comment (YinonFirstDAG) [reply dips/dip-158.md:121]: I update the same diagram in swimlanes. 
You can always check the URL below for the "latest and greatest" of the Charge payment (QR scan) flow
https://swimlanes.io/u/CVMdULSTu

Let's agree on the finalized version and then we can save as image, upload to GitHub and embed in the DIP

**Review by davidiw — APPROVED** [review]

**Pull request merged**
